cylc/flow/exceptions.py

```diff
@@ -83,24 +83,24 @@ def __init__(self, msg):
             self, 'Not loading %s (out of sequence bounds)' % msg)
 
 
-class ParamExpandError(SuiteConfigError):
+class ParamExpandError(WorkflowConfigError):
     """Exception for errors in Cylc parameter expansion."""
 
 
-class SuiteEventError(CylcError):
+class WorkflowEventError(CylcError):
     """Exception for errors in Cylc event handlers."""
 
 
-class SuiteServiceFileError(CylcError):
-    """Exception for errors related to suite service files."""
+class WorkflowServiceFileError(CylcError):
```

Comment:
`ServiceFileError` would be less lengthy, but I appreciate it might be a pain to change again

Reply:
Easy to do with a `perl` or `sed` one-liner.